sky/execution.py

```diff
@@ -1231,7 +1238,10 @@ def serve_down(
                 )
 
     try:
-        core.cancel(controller_cluster_name, all=True, _from_serve_core=True)
+        if handle is not None:
+            assert isinstance(handle, backends.CloudVmRayResourceHandle)
+            backend = backends.CloudVmRayBackend()
+            backend.cancel_jobs(handle, jobs=None)
```

Comment:
I think we still want to get usage for `serve_down`? this is as important as up

Reply:
This is `sky cancel` for controller & load balancer(redirector) job

Reply:
oh got it.